integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
So are we having an entire table just for activation codes? Would it be better to just store it in the user data table, with the keys being the user's ID and then "activationCode" and store it there?